R/forward_geocoding.R

```diff
@@ -102,7 +102,95 @@ oc_forward <-
     oc_check(res)
 
     # done!
-    oc_parse(res, output)
+    oc_parse(res, output, query = placename)
+  }
+
+#' @export
+oc_forward_df <-
+  function(data,
+           placename,
+           bind_cols = TRUE,
+           output = c("short", "all"),
+           key = oc_key(),
+           bounds = NULL,
+           countrycode = NULL,
+           language = NULL,
+           limit = 1,
+           min_confidence = NULL,
+           no_annotations = FALSE,
+           no_dedupe = FALSE,
+           no_record = FALSE,
+           abbrv = FALSE) {
+
+    placename <- data[[substitute(placename)]]
+    output <- match.arg(output)
+    add_request <- TRUE # Ensure that query column always exists
+    if (output == "short") {
+      no_annotations <- TRUE
```

Comment:
I am still not a big fan of 'silently' overriding parameters, even though it makes total sense to set `no_annotations` to `TRUE` here, after all, the annotations are not going to be in the "short" output. We could set the parameter `no_annotations = ifelse(output ==  "short", TRUE, FALSE)` to make it more transparent. It is a bit wordy, though, so it might not be the best solution. What do you think?